pkg/push/model_table.go

```diff
@@ -77,3 +83,53 @@ func NewColumn(name string, imp string) Column {
 
 func (c column) Name() string   { return c.name }
 func (c column) Import() string { return c.imp }
+
+type ImportedRow struct {
+	jsonline.Row
+}
+
+func (t *table) initTemplate() {
+	t.template = jsonline.NewTemplate()
+
+	if t.columns == nil {
+		return
+	}
+
+	if l := int(t.columns.Len()); l > 0 {
+		for idx := 0; idx < l; idx++ {
+			col := t.columns.Column(uint(idx))
+			key := col.Name()
+
+			switch col.Import() {
+			case "string":
+				t.template.WithString(key)
+			case "numeric":
+				t.template.WithNumeric(key)
+			case "base64":
+				t.template.WithBinary(key)
```

Comment:
With this change (only) :

```suggestion
				t.template.WithMappedBinary(key, []byte{})
```

And this configuration file 

```yaml
version: v1
tables:
  - name: "staff"
    keys:
      - "staff_id"
    columns:
      - name: "staff_id"
      - name: "picture"
        import: "base64"
```

The result is OK, but... not an actual solution because we shouldn't use `base64` value in `import:` parameter, but in `export:` parameter.